lib/xcov/coveralls_handler.rb

```diff
@@ -1,3 +1,4 @@
+require 'fileutils'
 require "tempfile"
```

Comment:
I still use the tempfile gem on line 57. I just give it the folder to create the temp file in instead of letting it choose. that way we always know we have write access